src/lib/y2network/sysconfig/interface_file.rb

```diff
@@ -158,9 +158,9 @@ def fetch(key)
       #
       # @todo Borrow logic from https://github.com/yast/yast-yast2/blob/6f7a789d00cd03adf62e00da34720f326f0e0633/library/network/src/modules/NetworkInterfaces.rb#L291
       #
-      # @return [Symbol] Interface's type depending on the file values
+      # @return [String] Interface's type depending on the file values
       def type
-        :eth
+        "eth"
```

Comment:
why is type a string here?

Reply:
well, reason is that in this case it can be also read from file, so I use value that can be specified in file as INTERFACETYPE . Or do you think it is bad approach?

Reply:
slightly misleading in the context of this pr, but makes sense in this case ;-)

Reply:
well, I need to change it as there is now basically only two representation short_name as string and InterfaceType as class. So no symbols anymore.